vignettes/quality_control_workflow.Rmd

```diff
@@ -0,0 +1,304 @@
+
+The input data should look something like that: 
+
+| Sample  |  Replicate |  Peptide  |  Condition | log2(intensity) |
+|:-------:|:----------:|:---------:|:----------:|:----------------:
+| sample1 |      1     |  PEPTIDER |   treated  |     14          |
+| sample1 |      1     |  PEPTI    |   treated  |     16          |
+| sample1 |      1     |  PEPTIDE  |   treated  |     17          |
+| sample2 |      1     |  PEPTIDER |  untreated |     15          |
+| sample2 |      1     |  PEPTI    |  untreated |     18          |
+| sample2 |      1     |  PEPTIDE  |  untreated |     12          |
+
+## Getting started
+
+Before we can start analysing our data, we need to load the **protti** package. This is done by using the base R function `library()`. In addition, we are also loading the packages `magrittr` and `dplyr`.
+
+```{r setup, message = FALSE}
+library(protti)
+library(magrittr)
+library(dplyr)
+
+```
+
+After having loaded our required packages, we can create a synthetic dataset, as it should be obtained though a treatment experiment with for example a protein, metabolite or small-molecule. 
+You can skip this step if you have your own data ready.
+
+We are creating a random dataset with 100 different proteins, out of which 5 % are significantly changing upon treatment. The dataset includes 3 replicates for 2 different conditions (treated and untreated).
+
+The `create_synthetic_data()` function is modeled after real LiP-MS data and its commonly observed data distributions.
+
+Please note that generally, quality control should be conducted on raw unfiltered data (the direct output of your search engine of choice).
+
+```{r create_synthetic_data}
+set.seed(123) # by setting the seed we are making sure that the random object generation can be reproduced
+
+data <- create_synthetic_data(
+  n_proteins = 100,
+  frac_change = 0.05,
+  n_replicates = 3,
+  n_conditions = 2,
+  method = "random_effect",
+  additional_metadata = TRUE
+)
+
+```
+
+## Quality control
+
+### Coefficients of variation
+
+A good first step to start with your quality control is to check if the coefficients of variation between your different replicates are in a reasonable range. Ideally, these should be below 15-20 %. If you see groups with higher CVs, there might have been a sample preparation error or the instrument performance was not ideal at the time of measurement. You can check if after `median_normalisation()` (depending on whether your search engine has performed median normalisation already) your CVs improve.
```

Comment:
```suggestion
A good first step of quality control is to check if coefficients of variation between replicates are in a reasonable range. Ideally, these should be below 15-20 %. If you see groups with higher CVs, there might have been a sample preparation error or the instrument performance was not ideal at the time of measurement. You can check if after `median_normalisation()` (depending on whether your search engine has performed median normalisation already) your CVs improve.
The "combined" group of CVs contains CVs across all samples and not only across the replicates of a certain condition. Ideally the combined CVs should always be the group with the highest CV.  
```

I would maybe also add clear formulas that show how the CV is calculated. E.g from intensity of a group (peptide/precursor/protein) over the replicates. and then just the formula. This will make people including Ludovic happy because they know exactly how we calculate stuff.